plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -142,6 +146,15 @@ public PluginCompatTester(PluginCompatTesterConfig config){
 	public PluginCompatReport testPlugins()
         throws PlexusContainerException, IOException, MavenEmbedderException
     {
+        File war = config.getWar();
+        if (war != null) {
+            populateSplits(war);
+        } else {
+            // TODO find a way to load the local version of jenkins.war acc. to UC metadata
+            splits = HISTORICAL_SPLITS;
```

Comment:
@jglick by using the jenkins-version.properties?

Reply:
Well we know the version number in this case (no `-war` argument), but we do not have an actual WAR (or `jenkins-core.jar`) handy. It is possible to get one using `mvn dependency:get` etc., just more work, and anyway the only users of this tool that I know of pass `-war`.